Packs/Alexa/Integrations/AlexaV2/AlexaV2.py

```diff
@@ -0,0 +1,219 @@
+        reliability=reliability,
+        score=score
+    )
+    domain_standard_context = Common.Domain(
+        domain=domain,
+        dbot_score=dbot_score
+    )
+    return domain_standard_context
+
+
+''' COMMAND FUNCTIONS '''
+
+
+def test_module(client: Client) -> str:
+    """Tests API connectivity and authentication'
+
+    Returning 'ok' indicates that the integration works like it is supposed to.
+    Connection to the service is successful.
+    Raises exceptions if something goes wrong.
+
+    Args (Client): a client to use
+
+    Return: 'ok' if test passed, anything else will fail the test.
+    """
+
+    try:
+        client.alexa_rank('google.com')
+        return 'ok'
+    except DemistoException as e:
+        if 'Forbidden' in str(e):
+            return 'Authorization Error: make sure API Key is correctly set'
+        raise e
+
+
+def alexa_domain(client: Client, args: Dict[str, Any]) -> List[CommandResults]:
+    domains = argToList(args.get('domain'))
+    if not domains:
+        raise ValueError('AlexaV2: domain doesn\'t exists')
+    command_results: List[CommandResults] = []
+    for domain in domains:
+        result = client.alexa_rank(domain)
+        domain_res = demisto.get(result,
+                                 'Awis.Results.Result.Alexa.TrafficData.DataUrl')
+        if domain_res == '404':  # Not found on alexa
+            raise DemistoException('Url cannot be found')
+        rank = demisto.get(result,
+                           'Awis.Results.Result.Alexa.TrafficData.Rank')
+        domain_standard_context: Common.Domain = rank_to_score(domain=domain_res,
+                                                               rank=arg_to_number(rank),
+                                                               threshold=client.threshold,
+                                                               benign=client.benign,
+                                                               reliability=client.reliability)
+
+        rank: str = rank if rank else 'Unknown'
+        result = {'Name': domain_res,
+                  'Indicator': domain_res,
+                  'Rank': rank}
+        table = {'Domain': domain_res,
+                 'Alexa Rank': rank,
+                 'Reputation': DBOT_SCORE_TO_TEXT.get(domain_standard_context.dbot_score.score, 'Unknown')}
+        readable = tableToMarkdown(f'Alexa Rank for {domain_res}', table, headers=list(table.keys()))
+        command_results.append(CommandResults(
+            outputs_prefix='Alexa.Domain',
+            outputs_key_field='Name',
+            outputs=result,
+            readable_output=readable,
+            indicator=domain_standard_context
+        ))
+    return command_results
+
+
+''' MAIN FUNCTION '''
+
+
+def main() -> None:
+    """main function, parses params and runs command functions
+
+    :return:
+    :rtype:
+    """
+    params = demisto.params()
+    api_key = params.get('credentials').get('password')
```

Comment:
use `demisto.get()`

Reply:
@ilaner don't forget this.

Reply:
```demisto.get()``` returns None if the value exists and it's value is None
In this situation we want to rank to be unkown, whether the path exists or not